app/components/action-feed/component.js

```diff
@@ -41,18 +40,18 @@ export default Component.extend({
         this.incrementProperty('page');
         const page = this.get('page');
         try {
-            const user = yield this.get('currentUser.user');
-            const actions = yield this.get('store').queryHasMany(user, 'actions', {
+            const actions = yield this.get('store').query('review-action', {
                 page,
-                embed: 'target',
+                embed: ['target'],
             });
             this.get('actionsList').pushObjects(actions.toArray());
             this.set(
                 'totalPages',
-                Math.ceil(actions.get('meta.total') / actions.get('meta.per_page')),
+                Math.ceil(actions.get('meta.pagination.total') / actions.get('meta.pagination.per_page')),
```

Comment:
Which version of the API uses `meta.pagination`? Is that new?

Reply:
That is a great question. When I was testing it out `meta.total` was incorrect but `meta.pagination.total` was an accurate number. I'll look into why it showed up.

Reply:
https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/serializers/osf-serializer.js#L143 (h/t @jamescdavis)

Reply:
`meta.total` is actually total pages.

Reply:
Ok, I think this is fine for now. If/when `totalPages` is available, we should use that instead.